src/main/java/com/baidu/hugegraph/structure/SchemaElement.java

```diff
@@ -38,12 +38,15 @@
     protected boolean checkExist;
     @JsonProperty("user_data")
     protected Map<String, Object> userdata;
+    @JsonProperty("status")
+    protected String status;
```

Comment:
add ignore-null to SchemaElement class:
@JsonInclude(JsonInclude.Include.NON_NULL)